lib/enkf/tests/enkf_queue_config.cpp

```diff
@@ -23,6 +23,36 @@ void test_empty() {
    queue_config_free(queue_config_copy);
 }
 
+void test_job_script() {
+  ecl::util::TestArea ta("script");
+  {
```

Comment:
Please remove the scopes and some of the white spaces when you are touching this code anyway ;)

Reply:
Also, you need to add it to the main body for it to be executed...